### PR TITLE
Completed html-forms exercise.

### PR DESCRIPTION
**Pull request by ChorongShin**

### CODE REVIEW

**Comment by ChorongShin** [comment]
Screen Shot 2022-01-25 at 2 58 27 PM

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <form>
+      <div>
+      <label for = "email">Email</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:9]: The equal signs for attributes in HTML should not have any spaces before or after them, please revise.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            <div>
+              <label for="comment">Comments</label>
+            </div>
+            <div>
+              <textarea id="comment" name="comment" rows="5" cols="40">
+              </textarea>
+              </div>
+              <div>
+                <input type="Submit" value="Submit">
+              </div>
```
Comment (codingcodymiller) [inline html-forms/index.html:45]: Your indentation started to get a bit wonky here 😬

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by ChorongShin** [comment]
Screen Shot 2022-01-25 at 3 08 27 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Still some indentation issues, please revise

**Comment by ChorongShin** [comment]
Screen Shot 2022-01-25 at 4 28 56 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Stella! This indentation is still pretty significantly off, please double-check that you ran the `npm ci` command in your `c0122-code-solutions` directory, the programs installed with that command should be helping you keep your formatting consistent.

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**